web/pdf_print_service.js

```diff
@@ -217,8 +217,8 @@ window.print = function print() {
   } finally {
     if (!activeService) {
       console.error('Expected print service to be initialized.');
-      if (OverlayManager.active === 'printServiceOverlay') {
-        OverlayManager.close('printServiceOverlay');
+      if (overlayManager.active === 'printServiceOverlay') {
```

Comment:
In line with comment below, and the existing code in `PDFPrintService.destroy`, we might want change this to the follow instead (to avoid an unlikely uncaught `Error`):
```javascript
ensureOverlay().then(function () {
  if (overlayManager.active !== 'printServiceOverlay') {
    return; // overlay was already closed
  }
  overlayManager.close('printServiceOverlay');
});
```
*Note:* I'd actually consider this a pre-existing issue with the code!